File: src/main/java/com/dochiri/hexagonal/application/user/port/in/LoginUserUseCase.java
package com.dochiri.hexagonal.application.user.port.in;

import com.dochiri.hexagonal.application.user.dto.LoginUserCommand;
import com.dochiri.hexagonal.application.user.dto.LoginUserResult;

public interface LoginUserUseCase {

    LoginUserResult login(LoginUserCommand loginUserCommand);

}

File: src/main/kotlin/com/dochiri/hexagonal/application/user/port/in/LoginUserUseCase.kt
package com.dochiri.hexagonal.application.user.port.`in`

import com.dochiri.hexagonal.application.user.dto.LoginUserCommand
import com.dochiri.hexagonal.application.user.dto.LoginUserResult

interface LoginUserUseCase {
    fun login(loginUserCommand: LoginUserCommand): LoginUserResult
}
